projects/isy-angular-widgets/src/lib/form-wrapper/form-wrapper.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import {createComponentFactory, Spectator} from '@ngneat/spectator';
 import {ReactiveFormsModule, FormControl, Validators} from '@angular/forms';
 import {FormWrapperComponent} from './form-wrapper.component';
 
-describe('FormWrapperComponent', () => {
+fdescribe('FormWrapperComponent', () => {
```

Comment:
f sollte entfernt werden
```suggestion
describe('FormWrapperComponent', () => {
```